documentation/learnmore/existing.md

```diff
@@ -51,6 +116,47 @@ against the empty databases (by altering the [url](/documentation/configuration/
 
 As soon as you add a new migration, it can be applied identically to any of your databases.
 
+### Example
+
+Looking at our instances, we have Production baselined and in line with our migration scripts. The Development instance has been cleaned and so is not in line. If we now run migrate against each instance it will align them. Migrate isn't strictly needed at this point for Production, however it will highlight that there is no work to be done as the baseline has been configured correctly.
+
+First we should migrate our Development instance.
+
+<pre class="console">&gt; flyway migrate -url="jdbc:postgresql://instances.spawn.cc:&lt;DevPort&gt;/pagila" \
+  -user="&lt;DevUsername&gt;" -password="&lt;DevPassword&gt;" -locations="filesystem:."</pre>
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:&lt;Port&gt;/pagila (PostgreSQL 11.0)
+Successfully validated 1 migration (execution time 00:00.087s)
+Creating Schema History table "public"."flyway_schema_history" ...
+Current version of schema "public": << Empty Schema >>
+Migrating schema "public" to version "1 - baseline migration"
+
+16 rows affected
+600 rows affected
+109 rows affected
+6 rows affected
+
+Successfully applied 1 migration to schema "public", now at version v1 (execution time 00:04.962s)</pre>
+
+As expected, Flyway recognised that the database has an empty schema and therefore created the `flyway_schema_history` table and ran our baseline migration. <i>I've removed some of the Postgres output for display purposes.</i>
+
+Now to migrate the Production instance.
+
+<pre class="console">&gt; flyway migrate -url="jdbc:postgresql://instances.spawn.cc:&lt;ProdPort&gt;/pagila" \
+  -user="&lt;ProdUsername&gt;" -password="&lt;ProdPassword&gt;" -locations="filesystem:."</pre>
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:&lt;Port&gt;/pagila (PostgreSQL 11.0)
+Successfully validated 1 migration (execution time 00:00.111s)
+Current version of schema "public": 1
+Schema "public" is up to date. No migration necessary.</pre>
+
+We can now see that our production database is already at version `1` which matches the `-baselineVersion` we specified during the `baseline` command. This means it is up to date with our migration scripts and nothing needs to be run.
+
+### Next steps
+
+If you have followed through this tutorial using our example and are ready to integrate Flyway into your own database, don't forget that [Spawn](https://spawn.cc) is available for free to create fast isolated copies for development and testing of your own databases. Enabling you to perfect the baselining process on a production like copy before the real thing. Find out how to create a data image of your own database [here](https://spawn.cc/docs/source-configuration-backup-postgres).
```

Comment:
`If you have followed our examples` instead of `If you have followed through this tutorial using our example`

`fast, isolated copies` instead of `fast isolated copies` (add a comma between them)

`of your own databases, enabling you to` instead of `of your own databases. Enabling you to`

`on a copy of Production before touching the real database` instead of `on a production like copy before the real thing`

`You can find out` instead of `Find out`

Reply:
Done